node/src/service.rs

```diff
@@ -147,11 +150,11 @@ where
 	let telemetry_span = TelemetrySpan::new();
 	let _telemetry_span_entered = telemetry_span.enter();
 
-  if parachain_config.offchain_worker.enabled {
```

Comment:
Why commented here? This would cause the offchain worker not activated properly.